gcc/fortran/scanner.c

```diff
@@ -2115,6 +2115,38 @@ preprocessor_line (gfc_char_t *c)
   current_file->line++;
 }
 
+/* Add LINE with length LEN and truncation status TRUNC to
+   FILE_CHANGES.  */
+static void
+add_line (gfc_char_t *line, int len, int trunc)
+{
+  gfc_linebuf *b;
+
+  b = XCNEWVAR (gfc_linebuf, gfc_linebuf_header_size
+		+ (len + 1) * sizeof (gfc_char_t));
+
+
+  b->location = linemap_line_start (line_table, current_file->line++, len);
+  /* ??? We add the location for the maximum column possible here,
+     because otherwise if the next call creates a new line-map, it
+     will not reserve space for any offset.  */
+  if (len > 0)
+    linemap_position_for_column (line_table, len);
+
+  b->file = current_file;
+  b->truncated = trunc;
+  wide_strcpy (b->line, line);
+
+  if (line_head == NULL)
+    line_head = b;
+  else
+    line_tail->next = b;
+
+      line_tail = b;
```

Comment:
I think this line has the wrong indentation.

Reply:
You're absolutely right.  I'm not sure how it was missed.

Reply:
I've updated my branch, which appears to automagically update the pull request.